dbt_subprojects/daily_spellbook/tests/aave/base/aave_base_interests_test.sql

```diff
@@ -0,0 +1,25 @@
+with unit_test1
+    as (select case
+                 when abs(variable_borrow_apy - 0.03485010188503055) / 0.03485010188503055 < 0.001
+                 then true
+                 else false
+               end as test
+        from   {{ ref('aave_v3_base_interest_rates' )}}
+        where  reserve = 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913
+               and hour = TIMESTAMP '2024-12-31 08:00'),
+    unit_test2
+    as (select case
+                 when abs(deposit_apy - 0.02788476319193648) / 0.02788476319193648 < 0.001
+                 then true
+                 else false
+               end as test
+        from   {{ ref('aave_v3_base_interest_rates' )}}
+        where  symbol = 'USDC'
```

Comment:
This test may not work properly.

Reply:
I updated this test to use correct values and a valid token symbol: https://github.com/duneanalytics/spellbook/pull/7404/commits/2f80416f3c5d351218f2d34fa4c048f1e9110fc3

Reply:
I updated these values to use real ones and the token symbol is now WETH instead: https://github.com/duneanalytics/spellbook/pull/7404/commits/2f80416f3c5d351218f2d34fa4c048f1e9110fc3